roles/openshift_aws/tasks/dns_route53_record.yml

```diff
@@ -0,0 +1,25 @@
+---
+- debug: msg="{{ openshift_aws_dns_records[l_openshift_aws_dns_element] }}"
+
+- name: querying elb
+  ec2_elb_facts:
+    names: "{{ openshift_aws_dns_records[l_openshift_aws_dns_element]['value'] }}"
+    region: "{{ openshift_aws_region }}"
+  register: elb_facts
+  when:
+    - "'elb' in openshift_aws_dns_records[l_openshift_aws_dns_element]['kind']"
+    - "'CNAME' in openshift_aws_dns_records[l_openshift_aws_dns_element]['type']"
+
+- name: creating record
+  route53:
+    command: create
+    overwrite: no
+    private_zone: "{{ (openshift_aws_dns_records[l_openshift_aws_dns_element]['scheme'] == 'internal') | ternary('yes','no') }}"
+    record: "{{ l_openshift_aws_dns_element }}.{{ openshift_aws_dns_zone }}"
+    type: "{{ openshift_aws_dns_records[l_openshift_aws_dns_element]['type'] }}"
+    ttl: 300
+    value: "{{ value | trim }}"
+    zone: "{{ openshift_aws_dns_zone }}"
+  vars:
+    value: "{% if 'CNAME' in openshift_aws_dns_records[l_openshift_aws_dns_element]['type'] -%} {{ elb_facts.elbs[0].dns_name }} \
```

Comment:
Please clarify why we're overriding the value here, and if type is not == CNAME, what do we get?